### ISSUE TITLE
New term request for Donkel- PMID 33717105

### ISSUE THREAD

**Issue by buniello** [EBISPOT/efo#1040]
For new term requests, please provide the following information:

## Preferred term label

protein-DNA complex measurement 

## Synonyms

NA

## Quantification of a macromolecular complex containing both protein and DNA molecules in a sample

the definition should be understandable even for non-specialists. Include a PubMed ID to refer to any relevant article that provides additional information about the suggested term.

## http://www.ebi.ac.uk/efo/EFO_0001444 or http://www.ebi.ac.uk/efo/EFO_0004747

Please look in the hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/efo)

## Attribution

If you would like a nanoattribution, please indicate your ORCID id

**Comment by buniello** [comment]
@kallia-p
no worries about the delay. Yes, that sounds like a good plan. I actually was not sure about the parent term in this case. 
Thank you for looking into that!

**Comment by kallia-p** [comment]
@buniello  Thanks for quick reply. @zoependlington what do you think about the two suggestions below? Any preference/caveats?

Suggestion 1 to add ‘myeloperoxidase (MPO)-DNA complex measurement’ Subclass of ‘macromolecular complex measurement’ while incrementally improving structure and axiomatisation in EFO: 



Import new term macromolecular complex from GO (requires manual import) http://purl.obolibrary.org/obo/GO_0032991
Definition from GO: A stable assembly of two or more macromolecules, i.e. proteins, nucleic acids, carbohydrates or lipids, in which the constituent parts function together.
In GO it is a subclass of cellular component in EFO it could be a Subclass Of cellular component and Subclass Of biological macromolecule
Equivalent To?

Assert protein, lipids, carbohydrates (and potentially other molecules in EFO) to be Subclass Of ‘biological macromolecule’ EFO:0004446 (Currently DNA and RNA are Subclasses of chemical entity and Subclasses of biological macromolecule but protein, lipid, carbohydrate are only Subclasses Of chemical entity)


Add new term in EFO macromolecular complex measurement
Equivalent To: measurement and is_about some macromolecular complex (but do we need to have classified all other candidate chemical entities under macromolecule or are we OK if we have classified proteins, lipids, carbohydrates, DNA and RNA under biological macromolecule)

Add new term myeloperoxidase (MPO)-DNA complex measurement Subclass of macromolecular complex measurement.


Suggestion 2 to add ‘myeloperoxidase (MPO)-DNA complex measurement’ Subclass of ‘macromolecular complex measurement’ while improving structure and axiomatisation in EFO by adding a few more terms (a bit more tricky with the axiomatisation/grouping though): 

Add new term from CHEBI macromolecule (dynamic import) https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:33839
Definition from CHEBI: A macromolecule is a molecule of high relative molecular mass, the structure of which essentially comprises the multiple repetition of units derived, actually or conceptually, from molecules of low relative molecular mass.
Could be Subclass of chemical entity. 
Equivalent to? 

Assert ‘biological macromolecule’ EFO:0004446 to be Subclass of macromolecule (because apart from biological macromolecules there are also synthetic biomolecules) 
Equivalent to?? macromolecule and has_role some biological role?

Import new term macromolecular complex from GO (can we import from GO?) http://purl.obolibrary.org/obo/GO_0032991
Definition from GO: A stable assembly of two or more macromolecules, i.e. proteins, nucleic acids, carbohydrates or lipids, in which the constituent parts function together.
In GO it is a subclass of cellular component in EFO it could be a Subclass Of cellular component and Subclass Of biological macromolecule
Equivalent To?


Assert protein, lipids, carbohydrates and   to be Subclass Of biological macromolecule (Currently DNA and RNA are Subclasses of chemical entity and Subclasses of biological macromolecule but protein, lipid, carbohydrate are only Subclass Of chemical entity)


Add new term in EFO macromolecular complex measurement
Equivalent To: measurement and is_about some macromolecular complex (but do we need to have classified many other chemical entities under 
macromolecule or are we OK if we have classified proteins, lipids, carbohydrates, DNA and RNA under biological macromolecule)

Add new term myeloperoxidase (MPO)-DNA complex measurement Subclass of macromolecular complex measurement.

**Comment by zoependlington** [comment]
@kallia-p 

We should keep the GO term as close to the domain as possible, so macromolecular complex should only be a child of cellular component in EFO like it is in GO. Eventually we will have a dynamic import of GO, too, but for now will require manual import.

Since the CHEBI dynamic import is next on my list, we shouldn't change the subclasses of CHEBI terms like DNA and RNA unless CHEBI also asserts them in the same way as we want to remain as aligned with CHEBI as possible and any changes may end up lost once the dynamic import is in place. I will try and make sure the import is available for our next release. Right now any CHEBI terms you would like to import would have to be imported manually.

**Comment by dosumis** [comment]
Suggestion:  Request term for NET from GO (CC).  Use this for linking/classification of measurement by biomarker.

**Comment by kallia-p** [comment]
@buniello @dosumis @zoependlington Thanks all for your suggestions. So as not to delay adding this term in because we need to work more on the parent term, I will add the specific measurement ‘myeloperoxidase (MPO)-DNA complex measurement’ as a subclass of 'measurement' for now and I will open another ticket for the parent term to be added later.

**Comment by buniello** [comment]
thank you @kallia-p
